securityconfig/opensearch.yml.example

```diff
@@ -37,10 +37,10 @@ opendistro_security.nodes_dn_dynamic_config_enabled: false
 opendistro_security.authcz.admin_dn:
   - "CN=kirk,OU=client,O=client,l=tEst, C=De"
 
-# Define how backend roles should be mapped to Open Distro Security roles
+# Define how backend roles should be mapped to Security roles
 # MAPPING_ONLY - mappings must be configured explicitely in roles_mapping.yml (default)
-# BACKENDROLES_ONLY - backend roles are mapped to Open Distro Security rules directly. Settings in roles_mapping.yml have no effect.
-# BOTH - backend roles are mapped to Open Distro Security roles mapped directly and via roles_mapping.yml in addition
+# BACKENDROLES_ONLY - backend roles are mapped to OpenSearch Security rules directly. Settings in roles_mapping.yml have no effect.
+# BOTH - backend roles are mapped to Security roles mapped directly and via roles_mapping.yml in addition
```

Comment:
Why some of the `Open Distro` are deleted, some are renamed to `OpenSearch`? Also `Security rules` seems to be a typo for `Security roles`.

Reply:
Updated